components/TextFieldsBeta/tests/snapshot/MDCBaseTextFieldTestsSnapshotTests.m

```diff
@@ -0,0 +1,62 @@
+// Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "MaterialSnapshot.h"
+
+#import <UIKit/UIKit.h>
+
+#import "MaterialTextFieldsBeta.h"
+
+@interface MDCBaseTextFieldTestsSnapshotTests : MDCSnapshotTestCase
+@end
+
+@implementation MDCBaseTextFieldTestsSnapshotTests
+
+- (void)setUp {
+  [super setUp];
+
+  // Uncomment below to recreate all the goldens (or add the following line to the specific
+  // test you wish to recreate the golden for).
+  //  self.recordMode = YES;
+}
+
+- (void)tearDown {
+  [super tearDown];
+}
+
+- (MDCBaseTextField *)createBaseTextField {
+  MDCBaseTextField *textField = [[MDCBaseTextField alloc] initWithFrame:CGRectMake(0, 0, 100, 60)];
+  textField.borderStyle = UITextBorderStyleRoundedRect;
+  return textField;
+}
+
+- (void)generateSnapshotAndVerifyForView:(UIView *)view {
+  UIView *snapshotView = [view mdc_addToBackgroundView];
+  [self snapshotVerifyView:snapshotView];
+}
+
+#pragma mark - Tests
+
+- (void)testTextFieldWithText {
+  // Given
+  MDCBaseTextField *textField = [self createBaseTextField];
+
+  // When
+  textField.text = @"Text";
+
+  // Then
+  [self generateSnapshotAndVerifyForView:textField];
+}
```

Comment:
Please add tests for each of the new APIs introduced in this PR.

Reply:
Right now the API for MDCBaseTextField only consists of `leadingView`, `trailingView`, `leadingViewMode`, `trailingViewMode`, and `mdc_adjustsFontForContentSizeCategory`. The only one that doesn't have tests is `mdc_adjustsFontForContentSizeCategory`, and it's not actually doing anything yet. Should I just remove it from this PR?

Reply:
+1 to removing it if you don't want to test its non-functioning behavior.